### PR TITLE
cleaning

### PR DESCRIPTION
**Pull request by jburel**
adjust display to match other guides

Nothing to test
cc @pwalczysko

### CODE REVIEW

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [OMERO-guides-push#14](https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-guides-push/14/). See the [console output](https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-guides-push/14/consoleText) for more details.
Possible conflicts:
  - PR #11 jburel 'cleaning'
    - docs/manual_analysis.rst
    - docs/headless_notebook.rst
    - docs/threshold_manual.rst

--conflicts

**Comment by pwalczysko** [comment]
Possibly not handled in this PR, but after you pushed the last commit about the notebook, I get in sphinx build:


```
/Users/pwalczysko/Work/omero-guide-fiji/docs/headless_notebook.rst:42: WARNING: Inline interpreted text or phrase reference start-string without end-string.
/Users/pwalczysko/Work/omero-guide-fiji/docs/headless_notebook.rst:74: WARNING: Inline interpreted text or phrase reference start-string without end-string.
```

**Comment by jburel** [comment]
I know what it is

File: docs/headless_notebook.rst [hunk 1]
```diff
@@ -43,9 +39,7 @@ See also
 
 #.  Open the *Attachments* harmonica in the right-hand panel and click on the plus icon. Browse and attach the :download:`fiji-macro-segment.ijm <../scripts/fiji-macro-segment.ijm>` to this Dataset as File Annotation.
 
-#.  Go to https://idr-analysis.openmicroscopy.org/training
-
-#.  Look under *Notebooks > Fiji* for *run_attached_macro.ipynb*.
+#.  Run the notebook `run_attached_macro.ipynb <https://mybinder.org/v2/gh/ome/omero-guide-fiji/master?filepath=notebooks/run_attached_macro.ipynb>.
```
Comment (pwalczysko) [inline docs/headless_notebook.rst]:
```suggestion
#.  Run the notebook `run_attached_macro.ipynb <https://mybinder.org/v2/gh/ome/omero-guide-fiji/master?filepath=notebooks/run_attached_macro.ipynb>`_.
```

Comment (pwalczysko) [reply docs/headless_notebook.rst]: Aha, you fixed it already in your last commit.

**Comment by jburel** [comment]
ok to merge it?

**Comment by jburel** [comment]
ok to merge @pwalczysko?

**Comment by pwalczysko** [comment]
Ok to merge

**Pull request merged**